src/_test_/commentCounter.test.js

```diff
@@ -0,0 +1,23 @@
+/** @jest-environment jsdom */
```

Comment:
Good job with adding tests but you need to mock the DOM environment instead of a giving an array. Kindly check the code example below and fix your tests.
```
const { JSDOM } = require('jsdom');

describe('CommentCounter', () => {
  let CommentsCounter;
  let dom;
  
  beforeEach(() => {
    dom = new JSDOM(`
      <!DOCTYPE html>
      <html>
        <body>
          <div id="comments">
            <p class="comment">Comment 1</p>
            <p class="comment">Comment 2</p>
            <p class="comment">Comment 3</p>
          </div>
        </body>
      </html>
    `);
    global.document = dom.window.document;
    global.window = dom.window;
    CommentsCounter = require('../CommentsCounter');
  });
  
  it('counts the number of elements with class "comment"', () => {
    const count = CommentsCounter();
    expect(count).toEqual(3);
  });
});
```